moveit_ros/visualization/motion_planning_rviz_plugin/src/motion_planning_frame.cpp

```diff
@@ -260,6 +260,12 @@ void MotionPlanningFrame::fillPlanningGroupOptions()
     ui_->planning_group_combo_box->addItem(QString::fromStdString(group_name));
 }
 
+void MotionPlanningFrame::setPlanningGroupText()
+{
+  const QSignalBlocker planning_group_blocker(ui_->planning_group_combo_box);
```

Comment:
I think, you don't need to block signals here. `setCurrentText` only sends a new signal, when the text actually changed.